node/pkg/fetcher/fetcher.go

```diff
@@ -188,7 +188,8 @@ func (a *App) startAllFetchers(ctx context.Context) error {
 func (a *App) stopFetcher(ctx context.Context, fetcher *Fetcher) error {
 	log.Debug().Str("adapter", fetcher.Name).Msg("stopping adapter")
 	if !fetcher.isRunning {
-		return errors.New("fetcher already stopped")
+		log.Debug().Str("adapter", fetcher.Name).Msg("fetcher already stopped")
```

Comment:
I think there might be a typo based on the code above (line 137)

```suggestion
		log.Debug().Str("fetcher", fetcher.Name).Msg("fetcher already stopped")
```

but then also line 189 should be updated.